.github/workflows/release.yml

```diff
@@ -134,12 +135,36 @@ jobs:
         run: |
           brew install postgresql
 
+      - name: Removed unused apps (MacOS arm64)
+        if: ${{ matrix.os == 'macos-latest-xlarge' && env.gcloud_archive_exist == '' }}
+        shell: bash
+        run: |
+          # MacOS arm64 runner only has 14GB avaialble, which is too small for our builds, so removing unused softwared.
+          df -hI /dev/disk3s1s1
+          sudo rm -rf /Applications/Xcode_14.3.1.app
+          sudo rm -rf /Applications/Xcode_15.0.1.app
+          sudo rm -rf /Applications/Xcode_15.1.app
+          sudo rm -rf /Applications/Xcode_15.2.app
+          sudo rm -rf /Applications/Xcode_15.3.app
```

Comment:
This might fail down the line if the runner changes - perhaps it would be better to change this to not fail the workflow?

Reply:
Makes sense!